packages/@apphosting/common/src/index.ts

```diff
@@ -1,5 +1,58 @@
 import { spawn } from "child_process";
 
+// Output bundle specifications to be written to bundle.yaml
+export interface OutputBundle {
+  version: "v1alpha";
+  serverConfig: ServerConfig;
+  metadata: Metadata;
+}
+
+// Fields needed to configure the App Hosting server
+interface ServerConfig {
+  // Command to start the server (e.g. ["node", "dist/index.js"]) assume this command is run from the root dir of the workspace
+  runCommand: string[];
+  // Environment variable present in the server execution environment.
+  environmentVariables?: EnvVarConfig[];
+  // The maximum number of concurrent requests that each server instance can receive.
+  // Defaults to 80.
```

Comment:
Do we want to drop these defaults now that we know GCP can adjust default values per user?

Reply:
What if we link to the product docs instead so we have a single source of truth?

Reply:
Removed the defaults in favor of linking the docs.